api_v3/lib/KalturaEntryService.php

```diff
@@ -964,13 +964,24 @@ protected function add(KalturaBaseEntry $entry, $conversionProfileId = null)
 			$templateEntry = entryPeer::retrieveByPK($conversionProfile->getDefaultEntryId());
 			if($templateEntry)
 			{
-				$dbEntry = $templateEntry->copyTemplate(true);
-				$dbEntry->save();
-			}
-			else
-			{
-				KalturaLog::err("Template entry id [" . $conversionProfile->getDefaultEntryId() . "] not found");
+				$entryType = $entry->type ? $entry->type : "null";
+				$apiTransformedType = kPluginableEnumsManager::coreToApi('entryType', $templateEntry->getType());
+				if($entryType != $apiTransformedType)
+					KalturaLog::debug("ENTRY_TEMPLATE_COPY_TYPE - original entry:template entry. type - ".$entryType.':'.$apiTransformedType);
+
+				$entryMediaType = "null";
+				if ($entry instanceof KalturaMediaEntry)
```

Comment:
Please move this logic into copyTemplate method.
Apply this rule also on other entry types, such as documentType in document entry, playlistType in playlist, externalSourceType in external media entry.